crates/pathfinder/src/bin/pathfinder/config.rs

```diff
@@ -1207,10 +1252,16 @@ mod tests {
     }
 
     #[test]
-    fn parse_versioned_constants_success() {
+    fn parse_versioned_constants_legacy() {
         super::parse_versioned_constants(
             "../executor/resources/versioned_constants_0_13_1_1.json".into(),
         )
         .unwrap();
     }
+
+    #[test]
+    fn parse_versioned_constants_success() {
+        super::parse_versioned_constants("resources/multi_versioned_constants.json".into())
```

Comment:
I'd rather put this file (used only for this particular test) into a `fixtures` directory because it's really just a test fixture.

Reply:
OK, but the same is true for `resources/invalid_versioned_constants.json`, no?

Reply:
Indeed, yes.